controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/deployment/DeploymentApiHandler.java

```diff
@@ -186,6 +188,13 @@ private HttpResponse root(HttpRequest request) {
                           jobStatus.pausedUntil().ifPresent(until -> jobObject.setLong("pausedUntil", until.toEpochMilli()));
                           jobStatus.coolingDownUntil(status.application().require(instance.instance()).change())
                                    .ifPresent(until -> jobObject.setLong("coolingDownUntil", until.toEpochMilli()));
+                          if (jobsToRun.containsKey(job)) {
+                              jobObject.setString("pending", jobsToRun.get(job).stream()
+                                                                      .allMatch(versions -> versions.sourcePlatform()
+                                                                                                    .map(versions.targetPlatform()::equals)
+                                                                                                    .orElse(true))
+                                                             ? "revision" : "upgrade");
```

Comment:
Use "application" and "platform" instead or is this something else?

Reply:
Hmm, ok.